### ISSUE TITLE
Accessibility tests failing in `main`

### ISSUE THREAD

**Issue by elycheea** [carbon-design-system/ibm-products#3177]
## What will this achieve?

Fix tests in CI.

## How will success be measured?

Tests should pass.

## Additional information

- Designs
- Existing code
- etc

https://github.com/carbon-design-system/ibm-products/actions/runs/5446628501/jobs/9908217875?pr=3176

**Comment by lee-chase** [comment]
@tay1orjones is this something Carbon has looked at before?

**Comment by elycheea** [comment]
Should double check if we need to address this for v1 as well before closing.

**Comment by tay1orjones** [comment]
I think y'all have seen by now, but we ran into the same. A new ruleset was published for accessibility-checker on July 1. If you're configured to use `latest`, it'll automatically pull the new ruleset potentially causing failures.
